### PR TITLE
Update Gradle to 6.3 (supporting Java 14)

### PR DESCRIPTION
**Pull request by SR-Lut3t1um**

### CODE REVIEW

**Review by gsmet — CHANGES_REQUESTED** [review]
Ah, I checked tomorrow and it wasn't released yet. Good to have it.

That being said, the PR is incomplete, see your previous one: https://github.com/quarkusio/quarkus/pull/7704/files .

We need to update everything.

Thanks!

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
@gastaldi looks good to me now but I will let you check it and merge it.

Thanks!

File: devtools/gradle/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,5 @@
-#Sat Mar 21 13:51:47 BRT 2020
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.2.2-all.zip
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-bin.zip
```
Comment (gastaldi) [inline devtools/gradle/gradle/wrapper/gradle-wrapper.properties]: Use the `-all.zip` version here. It helps when you need to browse the sources in IntelliJ

**Review by gastaldi — CHANGES_REQUESTED** [review]
Awesome, can you please squash your commits? Otherwise it looks good to me

**Review by gastaldi — APPROVED** [review]

**Pull request merged**